app/src/androidTest/java/org/mozilla/focus/activity/robots/BrowserRobot.kt

```diff
@@ -64,7 +69,7 @@ class BrowserRobot {
             return SearchRobot.Transition()
         }
 
-        fun clearBrowsingData(interact: HomeScreenRobot.() -> Unit): HomeScreenRobot.Transition {
+        fun clickTrashIcon(interact: HomeScreenRobot.() -> Unit): HomeScreenRobot.Transition {
```

Comment:
Just a small comment about naming, being more used to iOS UI, trashIcon is confusing to me, not totally explanatory about being for removing browsing data... but for closing tabs or other actions. But if that's totally clear on Android, then I'm fine with that :)

Reply:
sounds good, I'll change it back. Thanks!